webpack.config.js

```diff
@@ -42,61 +42,37 @@ const babelLoaderConfig = {
   },
 };
 
-const baseConfig = [
-  {
-    entry: './src/index-core.js',
-    output: {
-      path: path.join(__dirname, 'dist'),
-      filename: 'm3core.umd.js',
-      libraryTarget: 'umd',
-      library: 'm3core',
-    },
-    module: {
-      rules: [
-        eslintLoaderConfig,
-        babelLoaderConfig,
-      ],
-    },
+module.exports = {
+  mode: 'production',
+  entry: {
+    m3core: './src/index-core.js',
```

Comment:
_c.f._: comments above regarding bundle naming. Ultimately these dist bundles will the the published package for mirador as a library, so we want their names to be stable as we move through the versions. See https://www.npmjs.com/package/mirador

Reply:
It's also important that we be able to distinguish between umd builds and environment-specific builds, as well as minified and unminified builds.